README.md

```diff
@@ -159,6 +159,26 @@ platform](https://github.com/asreview/systematic-review-datasets).
 asreview data dedup benchmark:van_de_schoot_2017 -o van_de_schoot_2017_dedup.csv
 ```
 
+### Data Vstack (Experimental)
+Vertical stacking: combine as many datasets as you want into a single dataset.
+
+❗ Vstack is an experimental feature. We would love to hear your feedback.
+Please keep in mind that this feature can change in the future.
+
+Your datasets should be in any [ASReview-compatible data format](https://asreview.readthedocs.io/en/latest/data_format.html).
+All input files should be in the same format, the output path should also be of the same file format.
+
+Stack several datasets on top of each other: 
+```
+asreview data vstack output.csv MY_DATASET_1.csv MY_DATASET_2.csv MY_DATASET_3.csv
+```
+Here, 3 datasets are exported into a single dataset `output.csv`.
+The output path can be followed by any number of datasets to be stacked.
+
+#### Note
+Vstack does not do any deduplication.
+If you wish to create a single (labeled, partly labeled, or unlabeled) dataset from multiple datasets while having control over duplicates and labels, use compose instead.
```

Comment:
```suggestion
If you wish to create a single (labeled, partly labeled, or unlabeled) dataset from multiple datasets containing labeling decisions while having control over duplicates, use [compose](https://github.com/asreview/asreview-datatools#data-compose) instead.
```

Reply:
added this, although reference will only work when merged with PR #19